crates/youki/src/commands/features.rs

```diff
@@ -1,8 +1,203 @@
 //! Contains Functionality of `features` container command
+use std::path::Path;
+
 use anyhow::Result;
+use caps::{all, CapSet};
+use libcontainer::oci_spec::runtime::{
+    version, ApparmorBuilder, CgroupBuilder, FeaturesBuilder, IDMapBuilder, IntelRdtBuilder,
+    LinuxFeatureBuilder, LinuxNamespaceType, MountExtensionsBuilder, SelinuxBuilder,
+};
 use liboci_cli::Features;
 
+// Function to query and return capabilities
+fn query_caps() -> Result<Vec<String>> {
+    let mut available_caps = Vec::new();
+
+    for cap in all() {
```

Comment:
personal nitpick, for this particular case usign `caps::all` feels better as without the context of `use` statements above, the `all` fucntion name is confusing. I won't hold the PR on this being changed, but let me know your thoughts.

Reply:
> for this particular case usign caps::all feels better as without the context of use statements above, the all fucntion name is confusing.

That makes sense. I would also prefer `caps::all`. Let me fix it.

Reply:
fixed in eb5cd7e